pydoctor/astbuilder.py

```diff
@@ -953,10 +953,14 @@ def _get_all_args() -> Iterator[ast.arg]:
             yield from base_args.args
             varargs = base_args.vararg
             if varargs:
+                # Adding the asterixes
+                varargs.arg = f"*{varargs.arg}"
```

Comment:
shouldn't the asterixes be added in the view-rendering level?

For the low-level we try to keep the data as "pure" as possible.
Then, in the view level  / view-model / template we can decide how to show them.

Open option is to create a special class dedicated to variable or keyword arguments.
The class can wrap the actual name.

```suggestion
                varargs.arg = VariableArgument(varargs.arg)
```

Then, instead of checking if the name starts with '*' you can do an isinstance check...

but I guess that for this case, it's ok to have this convention and add the extra semantic via string characters and not via a type.

Reply:
I don't see how we could use a different type without having another massive change. I think it should be done while fixing #41, which is not tomorrow.

Reply:
Or maybe `VariableArgument` and `KeywordArgument` can be subclasses of `str`, in this case it could work!

Reply:
I'll give it a try